src/_base.scss

```diff
@@ -158,6 +142,10 @@ img {
 @if $shell-base-apply-bottom-margin-to-paragraphs {
```

Comment:
I get where you're coming from. I guess though we could do that for all opt in styles e.g.

```scss
@if $shell-base-apply-style-to-disabled-and-readonly-states {
    :disabled,
    [readonly] {
        opacity: $shell-base-disabled-and-readonly-state-opacity-strength;
    }
}
```

I guess there's part of me that still wants aggressive styles like that to remain un-opinionated for a higher reuse factor of a CSS library like Shell.

I mean we could say why have anything configurable in Shell? If you do that then is its existence warranted at all? It just lives in the CMDS CSS bundle. It's about how we see Shell's purpose.